application/github/v0/config/tasks.json

```
@@ -0,0 +1,1270 @@
+{
+  "$defs": {
+    "pull_request": {
+      "description": "A pull request object.",
+      "properties": {
+        "id": {
+          "description": "id of the PR",
+          "instillUIOrder": 1,
+          "title": "PR id",
+          "instillFormat": "integer",
+          "type": "integer"
+        },
+        "number": {
+          "description": "number of the PR",
+          "instillFormat": "integer",
+          "instillUIOrder": 2,
+          "title": "PR number",
+          "type": "integer"
+        },
+        "state": {
+          "description": "state of the PR",
+          "instillFormat": "string",
+          "instillUIOrder": 3,
+          "title": "PR state",
+          "type": "string"
+        },
+        "title": {
+          "description": "Title of the PR",
+          "instillFormat": "string",
+          "instillUIOrder": 4,
+          "title": "PR Title",
+          "type": "string"
+        },
+        "body": {
+          "description": "Body of the PR",
+          "instillFormat": "string",
+          "instillUIOrder": 5,
+          "title": "PR body",
+          "type": "string"
+        },
+        "diff_url": {
```

```suggestion
        "diff-url": {
```
Please make sure all property keys are in kebab-case.